File: GunDetectionApp/src/app/Models/user-alarm.ts
import { Guid } from 'guid-typescript';

export class UserAlarm {
  id: Guid;
  idAccount: Guid;
  name: string;
  lastName: string;
  email: string;
  phone: string;
  figth: boolean;
  gunDetected: boolean;
  personDetection: boolean;
  handsUp: boolean;
}

File: GunDetection.Models/UserAlarm.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace GunDetection.Models
{
    public class UserAlarm
    {
        public Guid   Id { get; set; }
        public Guid   IdAccount { get; set; }
        public string Name { get; set; }
        public string LastName { get; set; }
        public string Email { get; set; }
        public string Phone { get; set; }
        public bool  Figth          { get; set; }
        public bool GunDetected     { get; set; }
        public bool PersonDetection { get; set; }
        public bool HandsUp         { get; set; }
       
    }
}
